sql/catalyst/src/main/java/org/apache/spark/sql/connector/read/ScanBuilder.java

```diff
@@ -28,5 +28,11 @@
  */
 @Evolving
 public interface ScanBuilder {
+  enum orders { FILTER, AGGREGATE, COLUMNS };
+
+  // Orders of operators push down. Spark will push down filters first, then aggregates, and finally
+  // column pruning (if applicable).
+  orders[] PUSH_DOWN_ORDERS = {orders.FILTER, orders.AGGREGATE, orders.COLUMNS};
```

Comment:
We can simply document it in the classdoc.

Reply:
It seems like aggregate pushdown already includes column pruning. e.g., if we pushdown `max(c1), count(c2)`, the output should be the result of max, count, and the grouping columns. We can just say
```
If applicable (the operator is present and the source supports that operator), Spark pushes down filters to the source first, then pushes down aggregation or applies column pruning.
```